Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.InputSettings.cs

```diff
@@ -95,13 +95,21 @@ public float ControllerVerticalSensitivity
 
         public void DisableActions()
         {
+            _dashBehaviour?.DisableAction();
+            crouchInputAction?.Disable();
+            jumpInputAction?.Disable();
+            movementInputAction?.Disable();
```

Comment:
All input should be disabled

Reply:
Is it possible to just disable the base `InputActionAsset` here instead? That seems like a slightly more elegant and extendable approach, especially if we need to add new inputs

Reply:
oh wans't aware of that,i'll try it out.

Reply:
Actually. would that also disable the UI movement as well? especially for the controller?

Reply:
I could be off about this, but I believe the event system takes care of that separately and it should be safe, would be worth testing that first though. We could also rather than disabling the whole asset disable the player controls action map, and then any UI-specific controls would go in their own, new action map

Reply:
Cleaned it up to disable the Player Control action map